docs/sources/next/javascript-api/k6-experimental/browser/keyboard/press.md

```diff
@@ -1,6 +1,20 @@
 ---
 title: 'press(key[, options])'
-description: 'TODO'
+description: 'Dispatches a `keydown` event followed by an `keyup` event. This is useful for simulating pressing and releasing a key.'
 ---
 
 # press(key[, options])
+
+Dispatches a `keydown` event followed by an `keyup` event. This is useful for simulating pressing and releasing a key.
+
+| Parameter     | Type   | Default | Description                                                     |
+| ------------- | ------ | ------- | --------------------------------------------------------------- |
+| key           | string | `''`    | Name of the key to press, such as `Enter`.                      |
+| options       | object | `null`  | Optional settings.                                              |
+| options.delay | number | `0`     | Optional. Time to wait before pressing the key in milliseconds. |
```

Comment:
Is this the time to wait between `keyDown` and `keyUp`?

Reply:
It's the time before all those.

https://github.com/grafana/xk6-browser/blob/99686dfc84e86e0d02be70323a5cd4902c306438/common/keyboard.go#L243-L247

Reply:
Changed this here: [5ce2ca6](https://github.com/grafana/k6-docs/pull/1592/commits/5ce2ca67e8ae6f4b41fdee7621c6bd8ce7a61576)